shared/js/background/events.es6.js

```diff
@@ -185,8 +185,8 @@ const extraInfoSpec = ['blocking', 'responseHeaders']
 if (browser.webRequest.OnHeadersReceivedOptions.EXTRA_HEADERS) {
     extraInfoSpec.push(browser.webRequest.OnHeadersReceivedOptions.EXTRA_HEADERS)
 }
-// we determine if FLoC is enabled by testing for availability of its JS API
-const isFlocEnabled = ('interestCohort' in document)
+// we determine if browsingTopics is enabled by testing for availability of its JS API
+const isTopicsEnabled = ('browsingTopics' in document) && utils.isFeatureEnabled('browsingTopics')
```

Comment:
oh, we overlooked isFeatureEnabled before - does that mean that there was no way to disable it via config?

Reply:
Only for the header yeah.

Reply:
Specifically the header piece of this. The Content scope script would've been disabled as expected.